requirements-full.txt

```
@@ -17,7 +17,7 @@ opencv-python==4.5.1.48
 spikeextractors==0.9.10
 spikeinterface>=0.94.0
 neo==0.10.2
-roiextractors==0.4.17
+roiextractors @ git+https://github.com/catalystneuro/roiextractors.git@155efa5d0597529b5b02a8ada633d8b0452ab1d7
```

We did a release today, so now you can do this.
```suggestion
roiextractors==0.4.18
```